01-ruby-fundamentals/keyword-arguments.md

```diff
@@ -157,39 +205,38 @@ coffee_price(:drip, :tall, cold: true)
 coffee_price(:cappuccino, :grande)
 ```
 
-The only things we've changed are the parameters in the method signature, the body of the method remains the same. The end result should resemble [this gist](https://gist.github.com/droberts-ada/07714ad0e2a3bc5a60c50d6a78e02f15).
-
-There are a few things to note here:
-- In the method signature, each keyword argument comes with a _default value_
-  - The syntax is similar to keys and values in a hash
-- When you invoke a method, keyword arguments are optional
-  - If you omit one, the value from the method signature will be used instead
-- When a method is invoked, any keyword arguments are passed by name (a.k.a. _keyword_)
-- Keyword arguments can be passed in any order.
-- Keyword arguments can co-exist with regular arguments (also called _positional arguments_)
-  - Positional arguments must come first, both in the definition and when the method is called
-  - All positional arguments must be supplied for every call
+In each of these iterations, the only things we've changed are the parameters in the method signature, the body of the method remains the same. The end result should resemble [this gist](https://gist.github.com/droberts-ada/07714ad0e2a3bc5a60c50d6a78e02f15).
```

Comment:
We should probably see if we can make this Gist on an Ada account or port it to a current instructor.

Reply:
Happy to do that. I'm noticing this is just a copy/paste of what's in this file. Do we find it helpful to have the code in it's own file as well to make it easier to find or something?